CMakeLists.txt

```diff
@@ -187,7 +199,6 @@ else()
   )
 endif()
 
-add_alias(protobuf::libprotobuf libprotobuf)
```

Comment:
What's the reason to remove this line?

Reply:
The upstream protobuf CMakeLists.txt now does this. Some previous upgrade failed to notice this and it's just cruft now.